package.json

```diff
@@ -27,7 +27,7 @@
     "react-tween-state": "0.0.5"
   },
   "peerDependencies": {
-    "react-native": "^0.4.0"
+    "react-native": ">=0.4"
```

Comment:
I'm surprised `^0.4.0` didn't work.  The expected behaviour is to bump minor versions when needed, which should cover `0.5.0` and `0.6.0`.